addons/website_mail/security/website_mail_security.xml

```diff
@@ -2,6 +2,12 @@
 <odoo>
     <data noupdate="1">
 
+        <record id="mail_message_rule_public" model="ir.rule">
```

Comment:
There is some mismatch in the commits. One commit add a hardcoded domain, one commit adds a void file, and this one finishes the work by removing the hardcoded domain and adding stuff in the void file. I fear you will have to do strange stuff when rebasing. Bon courage.